_agencies/agency-international-development.html

```diff
@@ -1,6 +1,5 @@
 ---
 name: Agency for International Development
-data_url: https://s3-us-gov-west-1.amazonaws.com/cg-ff27f612-cd16-4a76-94f8-628986af2158/data/usaid
```

Comment:
I refactored how templates render data URLs so that the full domain name does not have to be replicated in each agency template. This will make the data URLs much more maintainable going forward.